packages/react-component-library/src/icons/index.ts

```diff
@@ -1,5 +1,9 @@
+import Bell from './Bell'
+import Graph from './Graph'
+import House from './House'
 import Search from './Search'
+import Tools from './Tools'
 import TriangleUp from './TriangleUp'
 import TriangleDown from './TriangleDown'
 
-export { Search, TriangleDown, TriangleUp }
+export { Bell, Graph, House, Search, Tools, TriangleDown, TriangleUp }
```

Comment:
Are you lifting these from prototype mocks? Are the designers creating these themselves or are we using an off the shelf icon library?

Reply:
The icon library question still seems to be up in the air